src/desktop/apps/articles/stylesheets/articles.styl

```diff
@@ -4,7 +4,7 @@ gutter-unit = 15px
 
 body.body-articles
   #main-layout-container
-    margin 50px 0px 0px 0px
+    margin 94px 0px 0px 0px
```

Comment:
@williardx - can we add a .md file or a comment in `NavBar` that includes the location of all of these hardcoded heights so that later, when we make this more dynamic / flexible, we'll know where to look? (A comment + link to this PR describing the problem would also suffice.)

Reply:
For sure, that's a good idea.

Reply:
@williardx, saw that this was merged - are you gonna follow up with another PR?

Reply:
Yeah I'm going to do further documentation separately